src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -71,23 +71,72 @@ class TezosNodeOperator(val node: TezosRPCInterface)(implicit executionContext:
     * @param accountIDs the ids
     * @return           the list of accounts wrapped in a [[Future]]
     */
-  def getAccountsForBlock(network: String, blockHash: BlockHash, accountIDs: List[AccountId]): Future[List[TezosTypes.Account]] =
+  def getAllAccountsForBlock(network: String, blockHash: BlockHash, accountIDs: List[AccountId]): Future[Map[AccountId, Account]] =
```

Comment:
This function no longer fetched all accounts for a block so it should be called `getAccountsForBlock()`. Perhaps we should keep the old `getAllAccountsForBlock()` functionality as well as it might be useful for future activities.

Reply:
Did the rename and restored the previous function to get all accounts for a block